src/widgets/DataTable/DataTable.js

```diff
@@ -61,7 +126,8 @@ DataTable.defaultProps = {
   getItemCount: items => items.length,
   initialNumToRender: 20,
   removeClippedSubviews: true,
-  windowSize: 11,
+  windowSize: 2,
```

Comment:
I'm all for it - just wondering the impact of the reduced windowSize and am currently unable to build and therefore view for myself.